throughput/tracker.go

```diff
@@ -61,14 +62,17 @@ func (t *tracker) logState(ctx context.Context, cli *jsonrpc.JSONRPCClient) {
 					if err != nil {
 						continue
 					}
+					currTime := time.Now()
+					diff := min(currTime.Sub(prevTime).Seconds(), 1)
 					utils.Outf(
 						"{{yellow}}txs seen:{{/}} %d {{yellow}}success rate:{{/}} %.2f%% {{yellow}}inflight:{{/}} %d {{yellow}}issued/s:{{/}} %d {{yellow}}unit prices:{{/}} [%s]\n", //nolint:lll
 						t.totalTxs,
 						float64(t.confirmedTxs)/float64(t.totalTxs)*100,
 						t.inflight.Load(),
-						current-psent,
+						uint64(float64(current-psent)/diff),
 						unitPrices,
 					)
+					prevTime = currTime
```

Comment:
Can we put all of these next to each other so that it's more obviously correct?

It's a bit strange to grab `current` before grabbing the lock, but then grab the current time after we've grabbed the lock. This could be much simpler if we just set them next to each other and update `psent` and `prevTime` after we've used them.